actuary.go

```diff
@@ -42,19 +43,40 @@ func (l *ContainerList) Running() bool {
 	return false
 }
 
-func CreateContainerList(c *client.Client) (l ContainerList) {
+type Target struct {
+	Client     *client.Client
+	Info       types.Info
+	Containers ContainerList
+}
+
+func NewTarget() (a Target, err error) {
+	a.Client, err = client.NewEnvClient()
+	if err != nil {
+		fmt.Printf("Unable to create Docker client")
```

Comment:
We should probably change all the printf's to:

`fmt.Printf("unable to create Docker client: %v\n", err)`

- starts lower case
- adds the error that we got
- adds a newline at the end.